### ISSUE TITLE
[Accessibility] Narrator cannot focus on RichTextBox with scrollbar in blue rectangle

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#10362]
### .NET version

9.0.100-alpha.1.23564.26

### Did it work in .NET Framework?

No

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

No, all reproduce in previous .Net versions

### Issue description

When tabbing a RichTextBox control with scrollbar, Narrator cannot focus on it with blue rectangle, but can announce it. It cannot repro for RichTextBox without scrollbar.

![image](https://github.com/dotnet/winforms/assets/26474449/207b0a5e-1d45-4f5f-9722-9b5b0f8e8d47)
![RichTextBox](https://github.com/dotnet/winforms/assets/26474449/2f4d4073-37c2-48aa-b2cb-680db286ef38)

### Steps to reproduce

Test application: 
[RichTextBoxSample.zip](https://github.com/dotnet/winforms/files/13446446/RichTextBoxSample.zip)

1. Build and run test application
2. Open Narrator, tab to RichTextBox with scrollbar to observe
